web/src/components/Navbar/index.js

```diff
@@ -70,20 +69,44 @@ class Navbar extends React.Component {
+              <defs>
+                <linearGradient id="paint0_linear" x1="25.6313" y1="23.249" x2="44.1932" y2="36.4364" gradientUnits="userSpaceOnUse">
+                  <stop stop-color="#E0E0E0" />
+                  <stop offset="1" stop-color="white" />
+                </linearGradient>
+                <linearGradient id="paint1_linear" x1="23.9127" y1="12.0471" x2="34.1044" y2="26.228" gradientUnits="userSpaceOnUse">
+                  <stop stop-color="#E5E5E5" />
```

Comment:
the dashed property names aren't going to work in React, kind of like when the illustration colors turned black.  Try to fix anything with dashes to camel-case like this:
```suggestion:
                  <stop stopColor="#E5E5E5" />
```